Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -86,13 +91,21 @@ internal final class PinchGestureHandler: GestureHandler, PinchGestureHandlerPro
         guard let view = gestureRecognizer.view else {
             return
         }
+
+        if panEnabled, focalPoint != nil {
+            Log.warning(
+                forMessage: "Possible pinch gesture recognizer misconfiguration: the specified focal point may be ignored when pinching. In order for the focal point to work, pinch pan has to be disabled.",
+                category: "Gestures")
+        }
+
```

Comment:
Could we move this warning to the GestureManager.options setter?

Reply:
That was my first place to put it :) I put it here to avoid having false positives in case user configures options one by one: 
```swift
mapView.gestures.options.focalPoint = .zero
mapView.gestures.options.pinchPanEnabled = false
```